namerd/main/src/main/scala/io/buoyant/namerd/DtabListHandler.scala

```diff
@@ -12,7 +12,7 @@ class DtabListHandler(
   override def apply(req: Request): Future[Response] =
     store.list().toFuture.map { list =>
       val response = Response()
-      response.contentType = MediaType.Html + ";charset=UTF-8"
```

Comment:
Same question here about why this is being removed

Reply:
See comment above.